frontend/packages/data-portal/app/routes/datasets.$id.tsx

```diff
@@ -1,24 +1,71 @@
 /* eslint-disable @typescript-eslint/no-throw-literal */
 
-import { Icon } from '@czi-sds/components'
-import { useLoaderData, useSearchParams } from '@remix-run/react'
 import { json, LoaderFunctionArgs } from '@remix-run/server-runtime'
 
 import { gql } from 'app/__generated__'
-import { GetDatasetByIdQuery } from 'app/__generated__/graphql'
 import { apolloClient } from 'app/apollo.server'
+import { DatasetMetadataDrawer } from 'app/components/Dataset'
+import { DatasetHeader } from 'app/components/Dataset/DatasetHeader'
 import { Demo } from 'app/components/Demo'
-import { Link } from 'app/components/Link'
-import { i18n } from 'app/i18n'
-import { cns } from 'app/utils/cns'
+import {
+  useCloseDatasetDrawerOnUnmount,
+  useDatasetDrawer,
+} from 'app/state/drawer'
 
 const GET_DATASET_BY_ID = gql(`
   query GetDatasetById($id: Int) {
     datasets(where: { id: { _eq: $id } }) {
-      id
-      deposition_date
+      # Dataset dates
       last_modified_date
       release_date
+      deposition_date
+
+      # Dataset metadata
+      id
+      title
+      description
+      funding_sources {
+        funding_agency_name
+      }
+
+      # TODO Grant ID
+      related_database_entries
+      dataset_citations
+
+      # Sample and experiments data
+      sample_type
+      organism_name
+      tissue_name
+      cell_name
+      cell_strain_name
+      # TODO cellular component
+      sample_preparation
+      grid_preparation
+      other_setup
+
+      # TODO add query for getting all unique affiliation names by dataset ID.
+      # For now we need to query all authors that have non-null affiliation
+      # names and then get unique values from the response.
+      authors(where: {affiliation_name: {_is_null: false}}) {
+        name
+        affiliation_name
+      }
```

Comment:
I have this implemented in my branch: https://github.com/chanzuckerberg/cryoet-data-portal/blob/ddb83f4188fbb087b43cc0096d481aff02e09a99/frontend/packages/data-portal/app/routes/datasets.%24id.tsx#L70-L73